medication_statement/medication_statement_output_1.json

```
@@ -0,0 +1,7 @@
+{
+    "title": "Zestril tablet 10mg",
+    "instructions": "1 maal per dag 1 tablet, oraal",
+    "prescribedBy": "Huisartsen, niet nader gespecificeerd",
+    "startDate": "2018-06-28",
+    "status": "active"
+}
```

Ik zou een array hier verwachten, aangezien er meerdere medication statements mogelijk zijn in de bundle.
```suggestion
[
    {
        "title": "Zestril tablet 10mg",
        "instructions": "1 maal per dag 1 tablet, oraal",
        "prescribedBy": "Huisartsen, niet nader gespecificeerd",
        "startDate": "2018-06-28",
        "status": "active"
    }
]
```